content/developer/blueprints/spec-relationships.md

```diff
@@ -7,11 +7,17 @@ weight: 700
 aliases: /blueprints/spec-relationships/
 ---
 
-`relationships` enable you to define how nodes relate to one another. For example, a `web_server` node can be `contained_in` a `vm` node or an `application` node can be `connected_to` a `database` node.
+`relationships` in Cloudify are syntactic elements in the DSL that establish dependencies between [node templates](../spec-node-templates), but also can be used to perform operations relevant to a particular node association.  Relationships are declared in a relationships section in a node template declaration at the same level as properties and interfaces, and are often defined in [plugins](../spec-plugins).  A node template can contain any number of relationship declarations, within certain constraints noted below.  Each relationship declaration refers to two node templates in a blueprint: the source and the target.  The source node is implicit; it is the node template that the relationship is declared in.  The target is explicit, and refered to by node id.
```

Comment:
My personal feeling is that this uses inaccessible language, which is unclear. I think extending the definition to include executable use case is useful, but the overall language may not be understandable in particular to ESL users.